src/mscorlib/shared/System/Diagnostics/Tracing/ActivityTracker.cs

```diff
@@ -247,11 +247,6 @@ public void Enable()
 
         #region private
 
-        /// <summary>
-        /// The current activity ID.  Use this to log normal events.  
-        /// </summary>
-        private Guid CurrentActivityId { get { return m_current.Value.ActivityId; } }
```

Comment:
@brianrob Could you please review the EventSource changes?

Reply:
Yup, thanks for pointing this out @jkotas.